### DISCUSSION TITLE
MUI text field rows property not working

### DISCUSSION THREAD

**Discussion by aadlc** [react-hook-form/react-hook-form#4729] [Q&A] [answered]
I am migrating a form to react-hook-form 7 but, the rows property on  `<TextField />` component is omitted.
Also, I forked React Hook Form - V7 - Controller from the sandbox and tried adding the property and still did not worked.
Here is the link to the forked sandbox: https://codesandbox.io/s/react-hook-form-v7-controller-forked-idlis?fontsize=14&hidenavigation=1&theme=dark

**Comment by jorisre** [comment, marked as answer]
You should add the `multiline` props to get `rows` working.

![Screenshot 2021-04-11 at 22 50 04](https://user-images.githubusercontent.com/7545547/114320853-44c1bd00-9b18-11eb-9076-511783c1adb9.png)

[MUI - TextField API](https://material-ui.com/api/text-field/#props)
